presto-main/src/main/java/com/facebook/presto/sql/planner/plan/SortNode.java

```diff
@@ -70,16 +62,10 @@ public PlanNode getSource()
         return source.getOutputSymbols();
     }
 
-    @JsonProperty("orderBy")
```

Comment:
I think you could keep this methods here, just for convenience. Then you would need to any usage of this node.